sbi/inference/base.py

```diff
@@ -7,7 +7,7 @@
 from copy import deepcopy
 from datetime import datetime
 from pathlib import Path
-from typing import Callable, Dict, List, Optional, Union, cast
+from typing import Callable, Dict, List, Optional, Union, cast, Tuple
```

Comment:
Run `isort` for ordering those imports

Reply:
I think this is resolved, it'd be good to mark it as such ;)